### PR TITLE
Remove broken links and var keywords

### LINKED ISSUE #2668: All JsPref links are broken
Hello, I had such a situation.
And I want to let you know too.

image

### PR DESCRIPTION
**Pull request by mrna0**
Closes #2668

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1847,14 +1847,14 @@ Other Style Guides
       return x + y;
     }
 
-    var x = 1;
-    var y = a + 2;
+    const x = 1;
```
Comment (ljharb) [inline README.md:1850]: this isn’t removing *every* var in the document - either they’re all fine, or they all need to go, but we shouldn’t do it partially.

File: README.md [hunk 2]
```diff
@@ -3907,16 +3907,14 @@ Other Style Guides
   - **MinnPost**: [MinnPost/javascript](https://github.com/MinnPost/javascript)
   - **MitocGroup**: [MitocGroup/javascript](https://github.com/MitocGroup/javascript)
   - **Muber**: [muber](https://github.com/muber/)
-  - **National Geographic**: [natgeo](https://github.com/natgeo/)
```
Comment (ljharb) [inline README.md:3910]: i believe this is now at https://github.com/natgeosociety

**Review by ljharb — CHANGES_REQUESTED** [review]

**Review by hossein13m — APPROVED** [review]

**Comment by mrna0** [comment]
Hi @ljharb, if there is something wrong my commits, I can fix it or cancel my  pull request.

**Review by slumericanB — APPROVED** [review]

**Comment by ljharb** [comment]
@mrna0 no, you’re fine - there’s just no rush. I’ll rebase it and land it soon.

File: README.md [hunk 3]
```diff
@@ -205,7 +205,7 @@ Other Style Guides
       value: 1,
 
       addValue: function (value) {
-        return atom.value + value;
+        return this.value + value;
```
Comment (ljharb) [inline README.md:208]: this is an incorrect change, since `this` will depend on the way `addValue` is called, but hardcoding `atom` will not.

**Review by ljharb — CHANGES_REQUESTED** [review]

**Review by ljharb — APPROVED** [review]

**Pull request merged**